### ISSUE TITLE
[easy] Reflection Warnings

### ISSUE THREAD

**Issue by amandhaasil** [plumatic/schema-generators#24]
I turned the `*warn-on-reflection*` to `true` and got two warnings from this library:

![Screen Shot 2022-03-11 at 09 23 51](https://user-images.githubusercontent.com/9124801/157867452-2f799bd2-81f5-47e3-baf5-4c0ae75e4a8e.png)

You can solve the one on line `src/schema_generators/generators.cljc:135` by adding `^long` like this:
```clojure
s/Inst (generators/fmap (fn [ms] (#?(:clj java.util.Date. :cljs js/Date.) ^long ms)) generators/int)}
```

And the one on `src/schema_generators/generators.cljc:157` just change `schema.core.EqSchema` to `schema.core.EnumSchema` like this:
```clojure
(defn enum-generators [s]
  (when (instance? schema.core.EnumSchema s)
    (let [vs (vec (.-vs ^schema.core.EnumSchema s))]
      (generators/fmap #(nth vs %) (generators/choose 0 (dec (count vs)))))))
```

Hope this helps

**Comment by gfredericks** [comment]
Fixed, thanks!

**Comment by gfredericks** [comment]
Released as `0.1.4`
